File: 01.aws.core/src/main/java/com/github/nkolytschew1/aws/core/mvc/controller/RedirectController.java
package com.github.nkolytschew1.aws.core.mvc.controller;

import org.springframework.stereotype.Controller;
import org.springframework.web.bind.annotation.RequestMapping;

/**
 * @author NKO
 * @version %I%, %G%
 * @since 1.X-SNAPSHOT
 */
@Controller
public class RedirectController {
	@RequestMapping("/")
	public String getStartPage(){
		return "forward:/index.html";
	}
}

File: 01.aws.core/src/main/kotlin/com/github/nkolytschew/aws/core/mvc/controller/RedirectController.kt
package com.github.nkolytschew.aws.core.mvc.controller

import org.springframework.stereotype.Controller
import org.springframework.web.bind.annotation.GetMapping

/**
 * @author NKO
 * @version %I%, %G%
 * @since 1.X-SNAPSHOT
 */
@Controller
class RedirectController {

  @GetMapping("/")
  fun getIndex() = "forward:/index.html"

}